main.go

```diff
@@ -219,6 +224,9 @@ func main() {
 	flag.StringVar(&deniedOutputPlugins, "fluent-bit-denied-output-plugins", "", "Comma separated list of denied output plugins even if allowUnsupportedPlugins is enabled. If empty, all output plugins are allowed.")
 	flag.IntVar(&maxLogPipelines, "fluent-bit-max-pipelines", 5, "Maximum number of LogPipelines to be created. If 0, no limit is applied.")
 
+	flag.BoolVar(&webhookEnabled, "validating-webhook-enabled", false, "Enable patching CA bindle for validating webhook")
```

Comment:
IMHO, the variable (and the flag) are misleading since the webhook is enabled regardless of whether the flag is set. Should be something about patching the CA. Could you also please remind me how we would use the flag?